src/components/Welcome/Content.tsx

```diff
@@ -0,0 +1,90 @@
+import { h, FunctionComponent } from 'preact'
+import classNames from 'classnames'
+
+import { buildIteratorKey } from '~utils'
+import { VIDEO_CAPTURE } from '~utils/constants'
+import { useLocales } from '~locales'
+import style from './style.scss'
+
+type InstructionsProps = {
+  className?: string
+}
+
+const Instructions: FunctionComponent<InstructionsProps> = ({ className }) => {
+  const { translate } = useLocales()
+
+  const instructionItemKeys = [
+    'welcome.list_item_doc',
+    'welcome.list_item_selfie',
+  ]
+
+  return (
+    <div className={classNames(style.instructions, className)}>
+      <span className={style.caption}>
+        {translate('welcome.list_header_doc_video')}
+      </span>
+      <ol>
+        {instructionItemKeys.map((itemKey) => (
+          <li key={itemKey}>{translate(itemKey)}</li>
+        ))}
+      </ol>
+    </div>
+  )
+}
```

Comment:
Will this screen only be for the Document ANSSI testing? Since I noticed the copy will not customisable by clients if this feature is enabled in an integration.

Reply:
From our discussion with Charlotte on Slack, it will be customisable and only visible if doc liveness enabled in integration. `MIGRATION` doc will need to be updated with the new keys added.